args: ['python3', 'tulip.py', './tests/compiler_errors/name_redefinitions/type_redef_type.tlp']
-------- stdout ----------
--------------------------
-------- stderr ----------
./tests/compiler_errors/name_redefinitions/type_redef_type.tlp:2:8 [ERROR]: Redefinition of Type `foo`.
--------------------------
exit code: 1
